galleries/tutorials/histogram_normalization.py

```diff
@@ -121,123 +145,148 @@
 # distribution function by both the length of the data and the width of the
 # bins:
 
-fig, ax = plt.subplot_mosaic([['False', 'True']], layout='constrained')
 dx = 0.1
 xbins = np.arange(-4, 4, dx)
-ax['False'].hist(xdata, bins=xbins, density=False, histtype='step', label='Counts')
 
+fig, ax = plt.subplot_mosaic([['False', 'True']], layout='constrained',
+                             figsize=(8, 4))
+
+
+ax['False'].hist(xdata, bins=xbins, density=False, histtype='step', label='Counts')
 # scale and plot the expected pdf:
-ax['False'].plot(xpdf, pdf * len(xdata) * dx, label=r'$N\,f_X(x)\,\delta x$')
-ax['False'].set_ylabel('Count per bin')
-ax['False'].set_xlabel('x bins [V]')
-ax['False'].legend()
+ax['False'].plot(xpdf, pdf * len(xdata) * dx, label=r'$N\,f_X(x)\,\delta x$', alpha=.5)
+
 
 ax['True'].hist(xdata, bins=xbins, density=True, histtype='step', label='density')
-ax['True'].plot(xpdf, pdf, label='$f_X(x)$')
-ax['True'].set_ylabel('Probability density [$V^{-1}$]')
-ax['True'].set_xlabel('x bins [$V$]')
+ax['True'].plot(xpdf, pdf, label='$f_X(x)$', alpha=.5)
+
+
+ax['False'].set(ylabel='Count per bin', xlabel='x bins [V]',
```

Comment:
Match labels with other Axes:
```suggestion
ax['False'].set(ylabel='Count per bin', xlabel='x bins [$V$]',
```